lib/index.js

```diff
@@ -167,6 +174,38 @@ I18N.prototype.configure = function( options ) {
 };
 
 I18N.prototype.setLocale = function( localeData ) {
+	if ( localeData && localeData[ '' ] && localeData[ '' ][ 'key-hash' ] ) {
+		var hash, hashLength, keyHash = localeData[ '' ][ 'key-hash' ];
+
+		if ( keyHash.substr( 0, 4 ) === 'sha1' ) {
+			hashLength = Number ( keyHash.substr( 5 ) );
+			hash = function( string ) {
+				var hash = sha1();
+				hash.update( string );
```

Comment:
Using a `var` name that appears in a higher scope could be confusing at first to those who don't know about javascript function contexts.

Reply:
Good call, changed.